chapter07/mdw/Ex04.pl

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use 5.010;
+
+foreach (<>) {
+  if ( /\b[^\s\\]*[A-Z]+[^\s\\]*[a-z]+[^\s\\]*\b/ || /\b[^\s\\]*[a-z]+[^\s\\]*[A-Z]+[^\s\\]*\b/ ) {
+ # comment as i have made this more complicated than needed, this will match any word with anything+capital+anything+non-capital+anything or the othe way around
+    print $_
+  }
```

Comment:
This is very complicated and is very difficult to understand. I think the easiest solution to this exercise is to do something similar to exercises 6 i.e. if ( /[a-z]/ && /[A-Z]/ ).